stable/stellar-core/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```

Comment:
add https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md

Reply:
I'm not sure I understand. Why do we need rbac in this chart? And if so: why in `Chart.yaml`?

Reply:
Coupling RBAC i a good practice, and allows for the addition of PodSecurityPolicies afterwards for example, enabling us to lock down the chart security-wise. If there are other reasons, i'm sure @davidkarlsen will speak up :-)

Reply:
It's in the best practices - and the best practice is to RBAC enable your cluster too - so that deployments can be limited in what actions to do. Did you read the link I provided?
I had to stick the comment somewhere - so since there were no rbac files I just stuck it into the Chart one...